src/opencast.js

```diff
@@ -490,10 +549,11 @@ const escapeString = s => {
   return new XMLSerializer().serializeToString(new Text(s));
 }
 
-const dcCatalog = ({ creator, title, seriesId }) => {
-  const seriesLine = seriesId
-    ? `<dcterms:isPartOf>${escapeString(seriesId)}</dcterms:isPartOf>`
-    : '';
+const dcCatalog = ({ creator, title, description, seriesID, username }) => {
+//  const seriesLine = seriesId
+//    ? `<dcterms:isPartOf>${escapeString(seriesId)}</dcterms:isPartOf>`
+//    : '';
```

Comment:
There was a reason why I did it like this. Sorry for not explaining it in a comment. I think Opencast returned a 400 bad request when the catalog would contain `<dcterms:isPartOf></dcterms:isPartOf>` (i.e. without value). So please again make this whole tag optional instead of only the value.

Reply:
I just double-checked it. Request works just fine with the isPartOf tag but without value. But its no problem to make it optional again.